blockrecord/header.go

```diff
@@ -216,3 +210,124 @@ func FoundationTxId(header *Header, digest blockdigest.Digest) merkle.Digest {
 	binary.LittleEndian.PutUint64(leBlockNumber, header.Number)
 	return merkle.NewDigest(append(digest[:], leBlockNumber...))
 }
+
+// AdjustDifficultyAtBlock - adjust difficulty at block
```

Comment:
I think it's much better if you comment the return values since multiple value is returned.

Reply:
updated